data/Mob.0.json

```diff
@@ -2101450,54 +2101750,54 @@
     "Z": 8785
   },
   {
-    "AggroLevel": 50,
-    "AggroRange": 100,
+    "AggroLevel": 75,
+    "AggroRange": 300,
     "BodyType": 1,
     "Brain": "",
-    "Charisma": 30,
+    "Charisma": 38,
     "ClassType": "DOL.GS.GameNPC",
-    "Constitution": 30,
-    "Dexterity": 30,
-    "Empathy": 30,
-    "EquipmentTemplateID": null,
-    "ExamineArticle": null,
+    "Constitution": 38,
+    "Dexterity": 38,
+    "Empathy": 38,
+    "EquipmentTemplateID": "",
+    "ExamineArticle": "",
     "FactionID": 0,
     "Flags": 0,
     "Gender": 0,
     "Guild": "",
-    "Heading": 807,
+    "Heading": 3045,
     "HouseNumber": 0,
-    "Intelligence": 30,
+    "Intelligence": 38,
     "IsCloakHoodUp": 0,
     "ItemsListTemplateID": null,
-    "LastTimeRowUpdated": "2000-01-01 00:00:00",
-    "Level": 8,
-    "MaxDistance": 0,
+    "LastTimeRowUpdated": "2023-10-24 17:22:03",
+    "Level": 9,
+    "MaxDistance": 3000,
     "MeleeDamageType": 2,
-    "MessageArticle": null,
+    "MessageArticle": "",
     "Mob_ID": "9ff4d993-00aa-48ef-a2ac-209d1de9c697",
     "Model": 102,
     "NPCTemplateID": 14889,
     "Name": "Blackfang",
     "OwnerID": "",
     "PackageID": "Public_DB",
-    "PathID": null,
-    "Piety": 30,
-    "Quickness": 30,
+    "PathID": "",
+    "Piety": 38,
+    "Quickness": 38,
     "Race": 0,
     "Realm": 0,
     "Region": 27,
     "RespawnInterval": 0,
     "RoamingRange": -1,
     "Size": 52,
-    "Speed": 180,
-    "Strength": 0,
-    "Suffix": null,
-    "TranslationId": null,
+    "Speed": 200,
+    "Strength": 110,
+    "Suffix": "",
+    "TranslationId": "",
```

Comment:
I'm seeing a lot of nulls becoming empty strings. Why is this?

Reply:
Mmm strange indeed I will check again as I didn’t modify this field in my testing DB

Reply:
Did you use DOLToolbox? I wonder if it is doing it.

Reply:
No I have used /mob create or copy directly in game

Reply:
Ok I've made a check on the tutorial region for my testing DB : 

In game : 
/mob create DOL.GS.GameNPC

In DB : 
SELECT m.Suffix, m.* FROM mob m WHERE Region=27 AND NAME='New Mob' AND Suffix=''
=> The query is returning the new mob meaning it's the current behavior 

Before my PR : 
SELECT COUNT(1) FROM mob m WHERE Region=27 AND Suffix IS null
=> 2602
SELECT COUNT(1) FROM mob m WHERE Region=27 AND Suffix=''
=> 382

After my PR : 
SELECT COUNT(1) FROM mob m WHERE Region=27 AND Suffix IS null
=> 2601
SELECT COUNT(1) FROM mob m WHERE Region=27 AND Suffix=''
=> 391

So only a portion of Suffix is blank. What do you mean by "a lot of nulls becoming empty strings" ? As I didn't notice a lot of them

Reply:
I think it's fine. I'll approve the PR. I think a followup task for myself could be to convert empty strings to null in the data.